roles/network_plugin/meta/main.yml

```diff
@@ -1,6 +1,7 @@
 ---
 dependencies:
   - role: network_plugin/cni
+    when: kube_network_plugin != 'calico'
```

Comment:
Can we also remove this change?

Reply:
I don't have a lot of bandwidth for this currently ; but feel free to use the PR as a base if it helps